iconservice/iconscore/icon_score_context.py

```diff
@@ -244,42 +240,72 @@ def update_dirty_prep_batch(self):
 
         self._tx_dirty_preps.clear()
 
-    def _update_elected_preps_in_term(self, dirty_prep: 'PRep'):
-        """Collect main and sub preps which cannot serve as a main and sub prep
-        to update the current term at the end of invoke
+    def _replace_prep(self, dirty_prep: 'PRep'):
```

Comment:
This function is called nowhere

Reply:
Removed _replace_prep() from IconScoreContext